reactivesearch/webpack.config.js

```diff
@@ -1,5 +1,6 @@
 const path = require('path');
 const {CleanWebpackPlugin} = require('clean-webpack-plugin');
+const NodePolyfillPlugin = require('node-polyfill-webpack-plugin');
```

Comment:
are we actually using NodePolyfillPlugin?

Reply:
It's one of the "solutions" for using the OS js Client class with newer versions of reactivesearch.